### PR TITLE
[SEED-909] Get identity using seed-services-client

### PR DESCRIPTION
**Pull request by alexmuller**
There are many similar cases in this file and across this project. This is the smallest possible change I can make to be sure I'm doing the right thing.

Use the API client to send requests to avoid duplicating code to talk to the seed-message-sender API.

### CODE REVIEW

File: hellomama_registration/utils.py [hunk 1]
```diff
@@ -6,6 +6,7 @@
 from django.conf import settings
 from registrations.models import Source
 from datetime import timedelta
+from seed_services_client import MessageSenderApiClient
 
 session = requests.Session()
 http = requests.adapters.HTTPAdapter(max_retries=5)
```
Comment (erikh360) [inline hellomama_registration/utils.py]: We add retries here

File: hellomama_registration/utils.py [hunk 2]
```diff
@@ -301,15 +302,9 @@ def get_messageset_schedule_sequence(short_name, weeks):
...
-    result = session.post(
-        url="%s/outbound/" % settings.MESSAGE_SENDER_URL,
-        data=json.dumps(payload),
-        headers={
-            'Content-Type': 'application/json',
-            'Authorization': 'Token %s' % (
-                settings.MESSAGE_SENDER_TOKEN,)
-        }
-    ).json()
+    ms_client = MessageSenderApiClient(settings.MESSAGE_SENDER_TOKEN,
```
Comment (erikh360) [inline hellomama_registration/utils.py]: We could send the session here.

**Review by erikh360 — COMMENTED** [review]
This looks good but it looks like we would be losing the  retry functionality. We could send the session to the client or update the client to always retry.

File: hellomama_registration/utils.py (hunk 2 above)
Comment (alexmuller) [reply hellomama_registration/utils.py]: Ah thanks! I've passed in the session. I think it'd probably be nicer for the API client to handle the retrying logic with a simple `retry` (or whatever) parameter but we I guess we can sort that out another time.

Comment (alexmuller) [reply hellomama_registration/utils.py]: That's exciting, passing in the session causes 15 tests to fail. Will investigate why!

Comment (erikh360) [reply hellomama_registration/utils.py]: I think it's a different session type or something, so passing in the session is maybe not a proper solution. We could pass in the correct one or just handle it properly in the client.

Comment (alexmuller) [reply hellomama_registration/utils.py]: praekeltfoundation/seed-services-client#33

**Comment by alexmuller** [comment]
Ok cool, I've changed this slightly to use an example of `IdentityStoreApiClient` to make sure I'm doing the right thing. I think this is ready for another review!

**Review by erikh360 — APPROVED** [review]
Looks good to me.

**Pull request merged**